jbpm-workitems/jbpm-workitems-rest/src/main/java/org/jbpm/process/workitem/rest/RESTWorkItemHandler.java

```diff
@@ -718,24 +717,28 @@ public RESTWorkItemHandler(String username,
                                String password,
                                ClassLoader classLoader,
                                boolean doCacheClient,
-                               int totalPoolConnections,
+                                int totalPoolConnections,
                                int maxPoolConnectionsPerRoute,
                                String handlingProcessId,
                                String handlingStrategy) {
-        this();
+        printProtocol();
         this.username = username;
         this.password = password;
         this.type = AuthenticationType.BASIC;
         this.classLoader = classLoader;
         this.doCacheClient = doCacheClient;
-        if (doCacheClient) {
+         if (doCacheClient) {
             connectionManager.setMaxTotal(totalPoolConnections);
             connectionManager.setDefaultMaxPerRoute(maxPoolConnectionsPerRoute);
         }
         this.handlingProcessId = handlingProcessId;
         this.handlingStrategy = handlingStrategy;
     }
 
+    private void printProtocol() {
```

Comment:
Please rename the method to something like `logProtocol` so it is obvious that it doesn't print anything on stdout.